libs/ui-watt/src/lib/components/input/shared/watt-picker-base.ts

```diff
@@ -388,6 +388,6 @@ export abstract class WattPickerBase
    * @ignore
    */
   private setSingleValue(value: string, input: HTMLInputElement) {
-    input.value = value;
+    input.value = !value ? '' : value;
```

Comment:
question: what would `value` be other than string?

Reply:
Just checked the setup that you're describing with the form control from search metering point. The reason for seeing `undefined` is because the form control's value is set to `undefined` in the `setInitialValue` method (the `q` route param is `undefined`). Is that a realistic setup though?

Reply:
I think it's better that we handle it (now we know it can actually happen), rather than just ignoring it and throw an error.

Reply:
Types should be adjusted now.